<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserFiles</class>
 <widget class="QWidget" name="UserFiles">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Padauk</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>MyDropBox</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>20</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>User  : </string>
   </property>
  </widget>
  <widget class="QLabel" name="user_lbl">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>20</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logout">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Log Out</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>601</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>50</number>
   </property>
   <property name="textDirection">
    <enum>QProgressBar::TopToBottom</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Padauk</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>My Files</string>
   </property>
  </widget>
  <widget class="QPushButton" name="upload_btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>601</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Share new file</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>291</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>60</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Padauk</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Shared With Me</string>
   </property>
  </widget>
  <widget class="QListWidget" name="sharedListWidget">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>90</y>
     <width>291</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>logout</sender>
   <signal>clicked()</signal>
   <receiver>UserFiles</receiver>
   <slot>logOut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>398</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>upload_btn</sender>
   <signal>clicked()</signal>
   <receiver>UserFiles</receiver>
   <slot>uploadFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>449</y>
    </hint>
    <hint type="destinationlabel">
     <x>8</x>
     <y>436</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>logOut()</slot>
  <slot>uploadFile()</slot>
 </slots>
</ui>
